deploy/pipeline/docs/tutorials/ppvehicle_attribute_en.md

```diff
@@ -0,0 +1,121 @@
+English | [简体中文](ppvehicle_attribute.md)
+
+# Attribute Recognition Module of PP-Vehicle
+
+Vehicle attribute recognition is widely used in smart cities, smart transportation and other scenarios. In PP-Vehicle, a vehicle attribute recognition module is integrated, which can identify vehicle color and model.
+
+| Task | Algorithm | Precision | Inference Speed | Download |
+|-----------|------|-----------|----------|---------------------|
+| Vehicle Detection/Tracking | PP-YOLOE | - | - | [Inference and Deployment Model](https://bj.bcebos.com/v1/paddledet/models/pipeline/mot_ppyoloe_l_36e_ppvehicle.zip) |
+| Vehicle Attribute Recognition | PPLCNet | 90.81 | 2.36 ms | [Inference and Deployment Model](https://bj.bcebos.com/v1/paddledet/models/pipeline/vehicle_attribute_model.zip) |
+
+
+Note:
+1. The prediction speed of the attribute model is based on the test of Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, the MKLDNN acceleration strategy is enabled, and the number of threads is 10.
+2. About the model PPLCNet, Please refer to [PP-LCNet Series](https://github.com/PaddlePaddle/PaddleClas/blob/release/2.4/docs/en/models/PP-LCNet_en.md).
+3. The training and test phase of vehicle attribute recognition model are both based on [VeRi dataset](https://www.v7labs.com/open-datasets/veri-dataset).
+
+
+- The published pretrained model support the recognition of 10 colors and 9 models, same with VeRi dataset. The detail are:
```

Comment:
The provided pre-trained model supports 10 colors and 9 models, which is the same with VeRi dataset. The details are as follows: